diff --git a/scripts/build-ffmpeg.py b/scripts/build-ffmpeg.py
index 1788e6b..7af752a 100644
--- a/scripts/build-ffmpeg.py
+++ b/scripts/build-ffmpeg.py
@@ -254,6 +255,7 @@ if not os.path.exists(output_tarball):
                 "--disable-examples",
                 "--disable-tools",
                 "--disable-unit-tests",
+                "--disable-neon",
             ],
         ),
         Package(
